models/pull.go

```diff
@@ -958,6 +958,100 @@ func (pr *PullRequest) AddToTaskQueue() {
 	})
 }
 
+// ChangeTargetBranch changes the target branch of this pull request, as the given user.
+func (pr *PullRequest) ChangeTargetBranch(doer *User, targetBranch string) (err error) {
```

Comment:
I would like move this function to `services/pull/`.

Reply:
I wasn't able to move everything into the service because of the comments.
But it does look cleaner now.